config/packages/prod/doctrine.yaml

```diff
@@ -1,5 +1,6 @@
 doctrine:
     orm:
+        auto_generate_proxy_classes: false
         metadata_cache_driver:
             type: service
```

Comment:
this is not working anymore in DoctrineBundle 2

We should sync the latest recipe changes for it:

https://github.com/symfony/recipes/blob/master/doctrine/doctrine-bundle/1.12/config/packages/prod/doctrine.yaml

Reply:
```bash
bin/console cache:clear --env=prod
```

should actually fail? maybe this can be part of the travis build?